src/ui/public/visualize/visualization.js

```diff
@@ -119,9 +119,7 @@ uiModules
             .then(() => {
             // renderComplete
               $scope.$emit('renderComplete');
-              if (!$scope.vis.visualizeScope) {
-                $el.trigger('renderComplete');
-              }
+              $el.trigger('renderComplete');
```

Comment:
the idea here was to only trigger `renderComplete` on the `<visualization>` element if the element was not just result of using `<visualize>` .... so i am guessing right now when using `<visualize>` and listening for renderComplete on it you will actually get two events fired (as this one will bubble up thru the DOM)

is there a reason to fire it in every scenario ?

Reply:
We used to have the `render-counter` attribute higher up the "component tree", so firing the renderComplete was appropriate there, but this PR moved this down so we now need this event fired lower.